planner/core/task.go

```diff
@@ -873,15 +873,22 @@ func (p *PhysicalStreamAgg) attach2Task(tasks ...task) task {
 
 // GetCost computes cost of stream aggregation considering CPU/memory.
 func (p *PhysicalStreamAgg) GetCost(inputRows float64, isRoot bool) float64 {
-	numAggFunc := len(p.AggFuncs)
-	if numAggFunc == 0 {
-		numAggFunc = 1
+	aggFactorSum := 0.0
+	for _, agg := range p.AggFuncs {
+		if fac, ok := aggFuncFactor[agg.Name]; ok {
+			aggFactorSum += fac
+		} else {
+			aggFactorSum += aggFuncFactor["default"]
+		}
+	}
+	if aggFactorSum == 0 {
+		aggFactorSum = 1.0
```

Comment:
So no agg func has more cost that single `bitXXX`? That seems weird.

Reply:
My opinion is that bit operation costs lower than other base operations.